app/controllers/people/person/positions/minister/edit.js

```diff
@@ -0,0 +1,61 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { dropTask } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import { combineFullAddress } from 'frontend-contact-hub/models/address';
+
+const FINANCING_CODE = {
+  SELF_FINANCED: '997073905f839ac6bafe92b76050ab0b',
+  FOD_FINANCED: '9d6f49b3d923b437ec3a91e8b5fa6885',
+};
+
+export default class PeoplePersonPositionsMinisterEditController extends Controller {
+  @service router;
+  @tracked willReceiveFinancing;
+  @tracked isCurrentPositionCheck;
+
+  get isCurrentPosition() {
+    return this.model.minister.agentEndDate === null;
```

Comment:
This check is currently not working as it should. The value will be undefined instead of null when it's not set and `null !== undefined` so it will return false when it shouldn't.

`return !this.model.minister.agentEndDate;` should work and covers both the case were it isn't set yet, or when the data is explicitly cleared again.